test/runtime/connectivity.go

```diff
@@ -48,230 +46,282 @@ var _ = Describe("RuntimeConnectivityTest", func() {
 		Expect(err).Should(BeNil())
 	}, 150)
 
+	dumpOutput := func(res *helpers.CmdRes) {
```

Comment:
Remove this function and its uses, it was used for debugging. Functionality if added should be added into the docker.* functions themselves